js/classes/entities/midground/enemies/village/bunny.js

```diff
@@ -78,6 +78,11 @@ class Bunny {
         const deathAnim = this.animations[this.base.getFacing()]["dying"];
         if (this.health <= 0 && deathAnim.currentFrame() === deathAnim.frameCount - 1) {
             this.removeFromWorld = true;
+            if (STORY.bunniesKilled) {
+                STORY.bunniesKilled++;
+            } else {
+                STORY.bunniesKilled = 1;
```

Comment:
Why are we setting bunniesKilled to 1?

Reply:
Cause it would be undefined, as we never initialize it to 0. If it has already been initialized tho it increments.